src/EFRepository/Repository.cs

```diff
@@ -219,5 +239,61 @@ protected virtual bool IsNew(TEntity entity)
 
 			return false;
 		}
+
+		public void StartTransaction(System.Data.IsolationLevel isolation)
+		{
+			throw new NotImplementedException();
+#if DOTNETFULL
+			DataContext.Database.BeginTransaction(isolation);
+#else
+			DataContext.Database.BeginTransaction();
+#endif
+		}
+
+		public void StartTransactionAsync(System.Data.IsolationLevel isolation, CancellationToken cancellationToken = default)

```

Comment:
These are also not implemented. Should we implement them or remove them for now? Thoughts?